src/leaflike/dev/sample_data.clj

```diff
@@ -0,0 +1,31 @@
+(ns leaflike.dev.sample-data
+  "Functions to generate and add dummy bookmarks. To be used for testing
+  and demoing."
+  (:require [leaflike.bookmarks.core :as bm-core]
+            [faker.internet :refer [domain-word]]
+            [faker.lorem :refer [sentences]]
+            [clojure.string :as str]))
+
+(defn gen-bookmark
+  "Generates a bookmark filled with a random title, URL and tags."
+  [& {:keys [num-tags] :or {num-tags 3}}]
+  (let [name (domain-word)
+        rand-sentence (first (sentences))
+        title (str name " - " rand-sentence)
+        tags (for [_ (range num-tags)] (domain-word))
+        tags-str (str/join "," tags)]
+    {:title title
+     :url (str "http://" name ".com")
+     :tags tags-str}))
+
+(defn add-bookmark
+  "Adds `bookmark` as `username`."
+  [username bookmark]
+  (bm-core/create {:params bookmark
+                   :session {:username username}}))
+
+(defn add-n-bookmarks
+  "Adds `num-bookmarks` generated bookmarks as `username`."
+  [username num-bookmarks]
+  (dotimes [i num-bookmarks]
```

Comment:
The index is not getting used so doseq might be preferable over dotimes

Reply:
I don't think `doseq` would be appropriate here since you'd be creating a seq (using `range`) and then iterating over that seq. You'd still be binding a variable that's not used. Instead in the new commit, I've stuck to `dotimes` and used the `_` binding.

Reply:
@samrat My bad, overlooked that you just need a looping construct here with no bindings that runs n times.
Anything would do really, so ```dotimes``` with the ```_``` is fine